modules/examples/eureka/modulestack.template

```diff
@@ -0,0 +1,26 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
If you do not need this stack, can you delete it?

Reply:
Updated